### PR TITLE
util: removed duplicate string trimming function

### PR DESCRIPTION
**Pull request by lpranam**
Signed-off-by: Pranam Lashkari <lpranam@collabora.com>
Change-Id: Ia150975467d1df8a9d05c8543bcb5d7369d29099

* Target version: master 

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: common/CommandControl.cpp [hunk 1]
```diff
@@ -35,9 +35,7 @@ void LockManager::generateLockedCommandList()
     std::string command;
     for (std::size_t i = 0; i < commandList.size(); i++)
     {
-        // just an extra check to make sure any whitespace does not sniff in command
-        // or else command will not be recognized
-        command = Util::trim_whitespace(commandList[i]);
```
Comment (Ashod) [inline common/CommandControl.cpp:40]: Should trimming still happen? Why not use `Util::trimmed()` ?

**Review by Ashod — COMMENTED** [review]
Thanks @lpranam. Why not keep the trimming?

File: common/CommandControl.cpp (hunk 1 above)
Comment (lpranam) [reply common/CommandControl.cpp:40]: I checked and `tokenize` does not include any space so its already trimmed no need to check again.

**Comment by eszkadev** [comment]
I will retrigger build as ssl cert is now ok for Jenkins and gtihub check will be able to work :)

**Review by Ashod — APPROVED** [review]

**Pull request merged**